### ISSUE TITLE
BadWindow on glfwPollEvents in BSPWM, Arch 64-bit

### ISSUE THREAD

**Issue by Mantissa-23** [glfw/glfw#1625]
OS: Arch x86-64
Release: 3.3.1
Error Messages: 
```
X Error of failed request:  BadWindow (invalid Window parameter)
  Major opcode of failed request:  40 (X_TranslateCoords)
  Resource id in failed request:  0x0
  Serial number of failed request:  171
  Current serial number in output stream:  171
```
Monitor Output: Black window briefly appears before crashing.

Program is included below (this is just the tutorial program without any rendering):

```C++
#include <glad/glad.h>
#include <GLFW/glfw3.h>
#include <iostream>

void framebuffer_size_callback(GLFWwindow* window, int width, int height);

int main(int argc, char** argv) {
  // Init
  glfwInit();

  // Set hints
  glfwWindowHint(GLFW_CONTEXT_VERSION_MAJOR, 3);
  glfwWindowHint(GLFW_CONTEXT_VERSION_MINOR, 3);
  glfwWindowHint(GLFW_OPENGL_PROFILE, GLFW_OPENGL_CORE_PROFILE);

  // Create window
  GLFWwindow* window = glfwCreateWindow(800, 600, "Wildshot", NULL, NULL);
  if (window == NULL) {
    std::cout << "Failed to create GLFW window" << std::endl;
    glfwTerminate();
    return -1;
  }
  glfwMakeContextCurrent(window);

  if(!gladLoadGLLoader((GLADloadproc)glfwGetProcAddress)) {
    std::cout << "Failed to initialize GLAD" << std::endl;
    return -1;
  }

  glViewport(0, 0, 800, 600);

  glfwSetFramebufferSizeCallback(window, framebuffer_size_callback);

  while(!glfwWindowShouldClose(window)) {
    glfwSwapBuffers(window);
    glfwPollEvents();
  }

  glfwTerminate();
  return 0;
}

void framebuffer_size_callback(GLFWwindow* window, int width, int height) {
  glViewport(0, 0, width, height);
}
```

The only observation I've been able to make so far is that:
- This crash only occurs when using bspwm. Some aspect of how bspwm manages windows causes the crash, however I don't know enough about Xorg and bspwm to figure out what.
- This crash does not occur when `glfwPollEvents();` is commented out.

**Comment by intrigus** [comment]
Duplicate of #1620 I guess.

**Comment by Mantissa-23** [comment]
Confirmed. I rolled back to 6b01aff, the parent of fe57e3c, and had no issue with the same executable. Closing.

**Comment by elmindreda** [comment]
@Mantissa-23 There is now a fix for this in `master` and `3.3-stable`.
